AEPCore/Sources/core/MobileCore.swift

```diff
@@ -125,8 +125,7 @@ public final class MobileCore: NSObject {
     /// - Parameter deviceToken: the device token for push notifications
     @objc(setPushIdentifier:)
     public static func setPushIdentifier(_ deviceToken: Data?) {
-        let hexString = SHA256.hexStringFromData(input: deviceToken as NSData?)
-        let data = [CoreConstants.Keys.PUSH_IDENTIFIER: hexString]
+        let data = [CoreConstants.Keys.PUSH_IDENTIFIER: deviceToken?.hexDescription ?? ""]
```

Comment:
We should add some e2e tests that cover this change, if not already present.

Reply:
We have tests here:
https://github.com/adobe/aepsdk-core-ios/blob/70d941c71e9d1b5fac6b525618cfdc4af4514d03/AEPCore/Tests/MobileCore%2BIdentityTests.swift#L78
and here:
https://github.com/adobe/aepsdk-core-ios/blob/70d941c71e9d1b5fac6b525618cfdc4af4514d03/AEPIntegrationTests/IdentityIntegrationTests.swift#L153